models/src/main/scala/coop/rchain/models/rholang/implicits.scala

```diff
@@ -127,11 +127,11 @@ object implicits {
     new Par(receives = Vector(r), locallyFree = r.locallyFree, connectiveUsed = r.connectiveUsed)
   def apply(n: New): Par =
     new Par(news = Vector(n),
-            locallyFree = NewLocallyFree.locallyFree(n),
+            locallyFree = NewLocallyFree.locallyFree(n, 0),
```

Comment:
I'd add a default value for the `depth` parameter to all changed signatures of `locallyFree`. This way, they wouldn't distract during review and would make it clear that 0 is the default value.

Reply:
> they wouldn't distract during review

because the call site wouldn't change in all the places like this one

Reply:
I prefer it to be explicit